### ISSUE TITLE
Invalid warning of unconfigured cops on new cops for  Style/HashEachMethods, Style/HashTransformKeys, Style/HashTransformValues

### ISSUE THREAD

**Issue by csalvato** [rubocop-hq/rubocop#7745]
I recently updated to rubocop 0.80.0 and am now seeing inaccurate warnings.

In my `.rubocop.yml`, I have the following configuration:

```yaml
Style:
  Enabled: false
```

But I am seeing a warning that I need to either enable or disable these cops:

```shell
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file:
 - Style/HashEachMethods
 - Style/HashTransformKeys
 - Style/HashTransformValues
Inspecting 82 files
..................................................................................
```

--------

## Expected behavior

Not present a warning for Style/HashEachMethods, Style/HashTransformKeys and Style/HashTransformValues when I disable all `Style`cops.

## Actual behavior

Seeing a warning for these new cops when all `Style` cops are disabled.

## Steps to reproduce the problem

1. Install rubocop
2. In `.rubocop.yml` disable all `Style` cops
3. Run rubocop
4. Observe warning

## RuboCop version

```
$ rubocop -V
0.80.0 (using Parser 2.7.0.2, running on ruby 2.6.5 x86_64-darwin19)
```

**Comment by geniou** [comment]
Have the same problem here.

**Comment by koic** [comment]
Thanks for the feedback. I opened the PR #7758.
